README.md

```diff
@@ -90,9 +90,9 @@ output = tokenizer.batch_decode(output_ids)
 #### GPU INT4
 ##### Install
 You may install **`bigdl-llm`** on Intel GPU as follows:
-> Note: See the [GPU installation guide](https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html) for more details.
+> Note: You should install GPU driver and oneAPI Base Toolkit beforehand. See the [GPU installation guide](https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html) for more details.
```

Comment:
Revert this? I think we don't need to say it here, since it is included in the doc page.

Reply:
I thought that it might be good to explicitly mention this in all readmes, in case users assume that the details in the install guide are optional, and only follow the pip install instructions in the readme.